TestCases/parallel_regression_AD.py

```diff
@@ -58,7 +58,7 @@ def main():
     discadj_cylinder3D.cfg_dir   = "disc_adj_euler/cylinder3D"
     discadj_cylinder3D.cfg_file  = "inv_cylinder3D.cfg"
     discadj_cylinder3D.test_iter = 5
-    discadj_cylinder3D.test_vals = [-3.724803, -3.838647, 0.000000, 0.000000] #last 4 columns
+    discadj_cylinder3D.test_vals = [-3.755976, -3.858734, 0.000000, 0.000000] #last 4 columns
```

Comment:
Yes because the case had a value of 0.95 specified (even if it did not do anything because the option did not apply to the discrete adjoint solver) I updated the residuals to keep the "feature" under regression, as it can break if e.g. solution old is set before/after.